openslides/utils/utils.py

```diff
@@ -1,6 +1,23 @@
+import re
 import roman
 
 
+first_re = re.compile('(.)([A-Z][a-z]+)')
+second_re = re.compile('([a-z0-9])([A-Z])')
```

Comment:
Maybe you could rename this two variables. Keep in mind, that we could add more regular expressions for other function in this file in the future.